io/eolearn/io/extra/geodb.py

```diff
@@ -4,7 +4,7 @@
 To use tasks from this module you have to install dependencies defined in `requirements-geodb.txt`.
 
 Credits:
-Copyright (c) 2017-2021 Matej Aleksandrov, Matej Batič (Sinergise)
+Copyright (c) 2021-2022 Matej Aleksandrov, Matej Batič (Sinergise)
```

Comment:
shouldn't this be 2017-2022?

Reply:
We actually only added this module last year so I wrote 2021-2022